conf/evolutions/default/221.sql

```diff
@@ -0,0 +1,12 @@
+# --- !Ups
+CREATE TABLE pano_history (
+   pano_id TEXT NOT NULL,
+   visited_timestamp TIMESTAMP,
+   pano_date TEXT NOT NULL,
+   location_current_pano_id TEXT NOT NULL,
+   PRIMARY KEY (pano_id),
+   FOREIGN KEY (location_current_pano_id) REFERENCES pano_history(pano_id)
+);
```

Comment:
Could you also add this line after the `CREATE TABLE` call? Trying to fix some issues with database permissions on the servers. Let me know if this causes any problems with your dev env!
```
ALTER TABLE pano_history OWNER TO sidewalk;
```